### DISCUSSION TITLE
Does OpenTelemetry supports a config through which one can filter spans based on duration of span

### DISCUSSION THREAD

**Discussion by MadhusudanN** [open-telemetry/opentelemetry-java-instrumentation#10073] [Q&A]
Hi All,
 I notice that we have this configuration option "otel.instrumentation.methods.include" exposed in the otel agent using which we can specify the methods for which we need spans.
Wanted to know if there any configuration through which we can filter spans based on duration of span. I notice there is a similar feature in elastic APM agent. (https://www.elastic.co/guide/en/apm/agent/java/current/config-core.html) 
![image](https://github.com/open-telemetry/opentelemetry-java-instrumentation/assets/26345035/11f484cc-a167-4d49-abcb-7fad5922c047)

Can someone please provide details whether open telemetry has support for such a configuration.

**Comment by laurit** [comment]
Opentelemetry java agent does not have support for this.
